src/scripts/background.js

```diff
@@ -52,11 +69,7 @@ window.TogglButton = {
     socket: null,
     maxRetryCount: 30,
     retryCount: 0,
-    retryInterval: 5,
-    get retryEnabled () {
-      return false;
-      // return TogglButton.websocket.retryCount < TogglButton.websocket.maxRetryCount;
-    }
+    retryInterval: 5
```

Comment:
In the case of the connection repeatedly working, then closing, then working, then closing..., we'd be retrying every 5s after the connection closes. IMO this is too soon.

Suggest retryInterval should be 30 seconds

Reply:
It is currently 5 + rand(0, 5) in the above case

Reply:
Would still rather look at e.g. 30s + rand(0, 60s), given that base 'retryInterval: 5' is current behavior & we are demonstrably causing problems for BE

Reply:
do you think

base retry: 15 seconds
max_jitter: 60 seconds

want to keep the regular user which might get disconnected on closing laptop lid happy

so yeah worst case time in first attempt

15 + 0 = 15 seconds after they open/resume their machines

in next attempt worst case is already 30 seconds now

15 * 2 + 0 = 30 seconds

in both cases assuming jitter is randomly 0